### ISSUE TITLE
Ace editor is not accessible to screen readers on macOS w/Chrome

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#5564]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.3.x
    OS Version      : macOS
    R Version       : Any

### Steps to reproduce the problem
On Mac, turn on a screen reader, open RStudio (using Chrome for Server, desktop of course is using Chromium), make sure screen-reader support is enabled under Help/Accessibility, and try to edit a file in the text editor.

### Describe the problem in detail
Various issues:
- use up/down arrow to move between lines, usually it will read the previous line, not the line you just moved to
- select text (e.g. Shift+arrow keys), nothing is said about the selection
- move right/left with arrow key, nothing is said, should read the individual characters

### Describe the behavior you expected
- minimally viable usage requires screen reader to speak a line when you move to it, notify when a selection is made, and read individual characters when you arrow left/right

The core problem is a known issue with Chromium/Chrome on macOS. Using Safari on macOS gives a somewhat usable experience.

https://bugs.chromium.org/p/chromium/issues/detail?id=912689

**Comment by gtritchie** [comment]
Current thinking is we will only support Safari for RStudio Server with VoiceOver on macOS for RStudio 1.3.
